py/formatfloat.c

```diff
@@ -100,15 +102,15 @@ static inline int fp_isless1(float x) {
 
 static const FPTYPE g_pos_pow[] = {
     #if FPDECEXP > 32
-    1e256, 1e128, 1e64,
+    MICROPY_FLOAT_CONST(1e256), MICROPY_FLOAT_CONST(1e128), MICROPY_FLOAT_CONST(1e64),
```

Comment:
These can only be double because they are such large numbers, but I guess for consistency it makes sense to have these macro wrappers.